pegasus/test/test_array.rb

```diff
@@ -1,7 +1,7 @@
 require_relative '../../lib/cdo/pegasus'
 require 'minitest/autorun'
 
-class ArrayTest < Minitest::Unit::TestCase
```

Comment:
oh, this change is actually wrong...i mistakenly ran Minitest 5.7 locally (forgot to `bundle exec` my `rake test`) and saw deprecation notices on these, but this was only renamed/deprecated in [Minitest 5.0.0](https://github.com/seattlerb/minitest/blob/master/History.rdoc#500--2013-05-10) (and our codebase is [locked to Minitest 4.x](https://github.com/rails/rails/blob/4-0-stable/activesupport/activesupport.gemspec#L26) in Rails 4.0.x). I'll change these back